contributor-orientation.php

```diff
@@ -396,124 +487,138 @@ function get_pages() {
+			'url'         => 'https://make.wordpress.org/openverse/handbook/new-contributor-guide/'
+		),
+/** 
+* Note: The Plugins, Meta and Tide teams are intentionally not currently displaying as suggested teams because they do not have sufficient new contributor onboarding processes 
+*/			
+		'plugins' => array(
+			'name'          => esc_html__( 'Get started with the Plugins Team', 'wporg' ),
+			'description'   => esc_html__( 'If you are a Plugin developer, subscribe to the Plugin review team blog to keep up with the latest updates, find resources, and learn about any issues around Plugin development.', 'wporg' ),
+			'icon'          => '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><rect x="0" fill="none" width="20" height="20"/><g><path d="M13.11 4.36L9.87 7.6 8 5.73l3.24-3.24c.35-.34 1.05-.2 1.56.32.52.51.66 1.21.31 1.55zm-8 1.77l.91-1.12 9.01 9.01-1.19.84c-.71.71-2.63 1.16-3.82 1.16H6.14L4.9 17.26c-.59.59-1.54.59-2.12 0-.59-.58-.59-1.53 0-2.12l1.24-1.24v-3.88c0-1.13.4-3.19 1.09-3.89zm7.26 3.97l3.24-3.24c.34-.35 1.04-.21 1.55.31.52.51.66 1.21.31 1.55l-3.24 3.25z"/></g></svg>',
+			'url'           => 'https://make.wordpress.org/plugins/handbook/',
+		),
+		'meta' => array(
+			'name'          => esc_html__( 'Get started with the Meta Team', 'wporg' ),
+			'description'   => esc_html__( 'The Meta Team makes WordPress.org, provides support, and builds tools for use by all the contributor groups. If you want to help make WordPress.org better, sign up for updates from the Meta blog.', 'wporg' ),
+			'icon'          => '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><rect x="0" fill="none" width="20" height="20"/><g><path d="M18 13h1c.55 0 1 .45 1 1.01v2.98c0 .56-.45 1.01-1 1.01h-4c-.55 0-1-.45-1-1.01v-2.98c0-.56.45-1.01 1-1.01h1v-2h-5v2h1c.55 0 1 .45 1 1.01v2.98c0 .56-.45 1.01-1 1.01H8c-.55 0-1-.45-1-1.01v-2.98c0-.56.45-1.01 1-1.01h1v-2H4v2h1c.55 0 1 .45 1 1.01v2.98C6 17.55 5.55 18 5 18H1c-.55 0-1-.45-1-1.01v-2.98C0 13.45.45 13 1 13h1v-2c0-1.1.9-2 2-2h5V7H8c-.55 0-1-.45-1-1.01V3.01C7 2.45 7.45 2 8 2h4c.55 0 1 .45 1 1.01v2.98C13 6.55 12.55 7 12 7h-1v2h5c1.1 0 2 .9 2 2v2z"/></g></svg>',
+			'url'           => 'https://make.wordpress.org/meta/handbook/about/contributor-day/',
+		),
+		'tide' => array(
+			'name'          => esc_html__( 'Get started with the Tide Team', 'wporg' ),
+			'description'   => esc_html__( 'Tide Team manages the Tide project, a series of automated tests run against every plugin and theme in the directory and displays PHP compatibility and test errors/warnings in the directory.', 'wporg' ),
+			'icon'          => '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><rect x="0" fill="none" width="20" height="20"/><g><path d="M17 7.2V3H3v7.1c2.6-.5 4.5-1.5 6.4-2.6.2-.2.4-.3.6-.5v3c-1.9 1.1-4 2.2-7 2.8V17h14V9.9c-2.6.5-4.4 1.5-6.2 2.6-.3.1-.5.3-.8.4V10c2-1.1 4-2.2 7-2.8z"/></g></svg>',
+			'url'           => 'https://make.wordpress.org/tide/feedback-support/',
+		),
```

Comment:
This was missing the final parenthesis, which caused a parse error on the site.

```suggestion
		),
	);
```

Reply:
GAH!